website/docs/spec/facets/dataset-facets/database_hierarchy.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 12
+---
+
+# Database Hierarchy Dataset Facet
+
+The facet contains information about the hierarchy of objects within database.
+
+Fields description:
+- `objectType`: Type of the object within the hierarchy (e.g. `TABLE`, `VIEW`, `TOPIC`).
+- `hierarchy`: Database hierarchy levels (e.g. `DATABASE` -> `SCHEMA` -> `TABLE`), from highest to lowest level. Array of objects:
```

Comment:
I think it's worth mentioning the order of the array is significant here

Reply:
Added a note